exercises/tournament/description.md

```diff
@@ -60,8 +60,3 @@ Devastating Donkeys;Courageous Californians;draw
 ```
 
 Means that the Devastating Donkeys and Courageous Californians tied.
-
-Your program should only process input lines that follow this format.
```

Comment:
This PR comment is inspired by the fact that one of the questions asked in https://github.com/exercism/xrust/issues/122 is "should I expect invalid lines?" despite the fact that invalid lines were in fact included in the test suite at the time (so the answer to the question was specified in the test suite)

The alternative to removing completely is a sentence saying "You need only process input lines that follow this format, as invalid lines will not be tested."

I very weakly support the addition of such a sentence, because:

One might consider this a useful addition if students will ask this question and it is useful to get an answer without looking at the test suite. Note that it is easier to figure out "there is an invalid input" (you only need to look for any one test case that has it) than "there are no invalid inputs" (you need to look through **all** test cases and confirm that everything is valid).

Wouldn't it be the case that forcing the student to look at every test case to answer that question is going against the one-test-at-a-time metholodogy? Even if we pretended this is a scenario where I am writing the tests from scratch (instead of having them be provided but not run, as various tracks do), that I would want to know whether I would have to deal with invalid tests ahead of time?

Side notes:

* I support removing "program" because #321.
* No opinion expressed on the one-test-at-a-time approach.